### DISCUSSION TITLE
Using log2(TPM/10+1) values in Seurat

### DISCUSSION THREAD

**Discussion by beginner984** [satijalab/seurat#4513] [Q&A] [answered]
Hi

I have downloaded `log2(TPM/10+1) values` of 10X Malignant ascites single cell from `GSE146026`
 as raw data is not available

Is there anyway to use this matrix and convert that to a `Seurat object`?

Thank you so much

**Comment by timoast** [comment, marked as answer]
Yes, run `CreateSeuratObject()` and skip the data normalization step since you're starting with normalized data

**Reply by jkrillbu** [reply]
@timoast The issue that I'm running into with this solution using Seurat 5.0 is that without running NormalizeData, other functions like ScaleData() don't work:
```
Error in `ScaleData()`:
! No layer matching pattern 'data' found. Please run NormalizeData and retry 
```
Is there an easy way to initialize a SeuratObject with the data layer?

It looks like I can just reassign the counts layer to the data layer and make the counts layer NULL, but I don't know if that will cause problems with downstream functions.
